frontend/apps/reader/modules/readerfont.lua

```diff
@@ -353,7 +353,7 @@ end
 function ReaderFont:getFontSettingsTable()
     local settings_table = {}
 
-    if Device:isAndroid() or Device:isDesktop() or Device:isEmulator() then
+    if Device:isAndroid() or Device:isDesktop() or Device:isEmulator() or Device:isPocketBook() then
```

Comment:
I'd say it's starting to look like it's time for some kind of Device:hasSystemFonts() abstraction but I suppose it's just this one file?

Reply:
Exactly. But the same happens with Device:hasExitOptions, which is only used on reader and FM menu.

Reply:
Either way's fine by me for now. ;-)